blocks/media-gallery/media-gallery.css

```diff
@@ -0,0 +1,160 @@
+.media-gallery-wrapper {
+    width: 100vw;
+}
+.media-gallery {
+    display: grid;
+    grid-gap: 1rem;
+    width: calc(100vw - 4rem);
+    margin: auto
+}
+
+.media-gallery p {
```

Comment:
I'd suggest using css nesting, let's let @tim-white-esri confirm that's fine with him (support is pretty widespread already: https://caniuse.com/css-nesting)

Reply:
Fine with me. I haven't used native CSS nesting yet; looking forward to trying it out.